CHANGELOG.md

```diff
@@ -1,5 +1,73 @@
 # CHANGELOG
 
+## Release 2.4.4 (in development)
+
+### Breaking changes
+- Due to a critical vulnerability with CVSS score of 10.0 that was discovered
```

Comment:
Can we add a link to the CVE(s) and to the PRs fixing them?

Reply:
Added